src/ert/gui/tools/manage_experiments/storage_info_widget.py

```diff
@@ -196,6 +197,51 @@ def _currentItemChanged(
             tuple(self._ensemble.get_realization_list_with_responses()),
         )
 
+        scaling_ds = self._ensemble.load_observation_scaling_factors()
+
+        def _render_scaled_obs() -> None:
+            assert scaling_ds is not None
+            _obs_df = observation_ds.to_dataframe().reset_index()
+            # Should store scaling by response type
+            # and use primary key for the response to
+            # create the index key.
+            index_cols = list(
+                set(observation_ds.to_dataframe().reset_index().columns)
+                - {
+                    "observations",
+                    "std",
+                }
+            )
+
+            # Just to ensure report step comes first as in _es_update
+            if "report_step" in index_cols:
+                index_cols = ["report_step", "index"]
+
+            # for summary there is only "time"
+            index_key = ", ".join([str(_obs_df[x].values[0]) for x in index_cols])
+            scaling_factors = (
+                scaling_ds.sel(obs_key=observation_name, drop=True)
+                .sel(index=index_key, drop=True)["scaling_factor"]
+                .dropna(dim="input_group")
+                .values
+            )
+
+            cumulative_scaling: float = (
+                reduce(lambda x, y: x * y, scaling_factors) or 1.0
+            )
+
+            original_std = observation_ds.get("std")
+            assert original_std
```

Comment:
Is it possible to avoid this naked assert?

Reply:
It gives this mypy error on `yerr=original_std * cumulative_scaling` below:
```
src/ert/gui/tools/manage_experiments/storage_info_widget.py:240: error: Unsupported operand types for * ("None" and "float")  [operator]
src/ert/gui/tools/manage_experiments/storage_info_widget.py:240: note: Left operand is of type "DataArray | None"
```

and we expect it to be not `None` so I think it should be ok. Could do a `#type: ignore` instead also though